src/SceneViewEdit/README.md

```diff
@@ -96,6 +95,3 @@ Only one draw or edit operation can be active at the time and preivous is cancel
 
 <img src="screenshot1.png"/>
 <img src="screenshot2.png"/>
-
-[](Esri Tags: ArcGIS Runtime SDK .NET WinRT WinStore WPF WinPhone C# C-Sharp DotNet XAML MVVM)
-[](Esri Language: DotNet)
```

Comment:
Are these not in use by our search engines any longer?

Reply:
No. There was an effort a few years back to remove them from the samples IIRC.